File: AnagramSolver.Contracts/Models/SearchInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AnagramSolver.Contracts.Models
{
    public class SearchInformation
    {
        public string UserIp { get; set; }
        public DateTime SearchTime { get; set; }
        public string SearchedWord { get; set; }
        public string Anagram { get; set; }
    }
}

File: AnagramSolver.SinglePageApp/Models/SearchInformation.js
class SearchInformation
{
    id;
    userIp;
    searchTime;  
    searchedWord;
    anagram;
    
  constructor(id, userIp, searchTime, searchedWord, anagram) {
        this.id = id
        this.userIp = userIp
        this.searchTime = searchTime
        this.searchedWord = searchedWord
        this.anagram = anagram
  }
}